src/components/Slide/SlideBase/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import { Slide as MaterialSlide, SlideProps as MaterialSlideProps } from "@material-ui/core";
+
+export interface SlideBaseProps extends MaterialSlideProps {}
+
+const SlideBase = React.forwardRef((props: SlideBaseProps, ref) => {
+    return <MaterialSlide ref={ref} {...props} mountOnEnter unmountOnExit />;
```

Comment:
I think we should give other devs ways to customize mount behavior

Reply:
![image](https://user-images.githubusercontent.com/58205909/131536537-19148155-9c3f-4770-af9a-47058b5ec2c5.png)